neptune/new/internal/artifacts/file_hash.py

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (c) 2021, Neptune Labs Sp. z o.o.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+import typing
+from pathlib import Path
+from dataclasses import dataclass
+
+from datalite import datalite
```

Comment:
have you used that? I doesn't seem to have much traction (3 stars on GH...)

Reply:
No, I don't. It should have more than 3 stars :wink: